das/daser.go

```diff
@@ -264,6 +263,8 @@ func (d *DASer) checkForByzantineError(ctx context.Context, hash []byte, height
 		err := d.bcast.Broadcast(ctx, fraud.CreateBadEncodingProof(hash, height, byzantineErr))
 		if err != nil {
 			log.Errorw("fraud proof propagating failed", "err", err)
+			return
 		}
+		log.Warn("fraud detected and broadcasted successfully. Shutting down services…")
```

Comment:
I would log.Warn("Shutting down services...") regardless of whether there was an error broadcasting or not, no?